### PR TITLE
Checkpoint completed

### PR DESCRIPTION
**Pull request by sayidu**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
+    /* eslint-disable */
+    function applyFunction() {
+      let testString = document.getElementById("testString").value;
+      let selectMethod = document.getElementById("selectMethod").value;
+      let error = '';
+      let result;
+      let methodDetails = '';
```
Comment (andela-kakpobome) [inline script.js:7]: ### Critical

Unnecessary use of `let`

File: src/string.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    return this.words().length;
+  },
+
+  /**
+   * toCurrency()
+   * Returns a currency representation of the String
+   * @returns {String}
+   */
+  toCurrency() {
+    return this.replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,');
```
Comment (andela-kakpobome) [inline src/string.js:75]: ### Critical

Unnecessary capture groups declared for lookahead operations.

File: script.js (hunk 1 above)
Comment (andela-kakpobome) [reply script.js:7]: @andela-sayidu has this now been resolved but is somehow not reflecting here 😕 ?

Comment (sayidu) [reply script.js:7]: @andela-kakpobome I just addressed this with the PR:  https://github.com/andela-sayidu/string-class/pull/4/files

**Pull request merged**